src/main/cluster-manager.ts

```diff
@@ -12,14 +12,14 @@ export class ClusterManager extends Singleton {
   constructor(public readonly port: number) {
     super();
     // auto-init clusters
-    autorun(() => {
-      clusterStore.enabledClustersList.forEach(cluster => {
+    reaction(() => clusterStore.enabledClustersList, (clusters) => {
```

Comment:
I can slice this to a separate PR if needed. Autorun causes side-effects because it triggers on "any" change related to enabled clusters. Reaction only runs when the list itself changes. 

Caused massive headache when adding a cluster suddenly stopped working after I removed a cluster kubeconfig from a filesystem (while testing this PR locally).